src/AzureAuth/Commands/Info/CommandInfoResetDeviceID.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Commands.Info
+{
+    using System.IO.Abstractions;
+    using McMaster.Extensions.CommandLineUtils;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Office.Lasso.Telemetry;
+
+    /// <summary>
+    /// The command <see cref="CommandResetDeviceID"/> reset the device ID in local storage.
+    /// </summary>
+    [Command(Name = "reset-device-id", Description = "Reset your device identifier.")]

```

Comment:
```suggestion
    [Command(Name = "reset-device-id", Description = "Reset your AzureAuth telemetry device identifier.")]

```
Only saying deveice identitifer may scare someone as that could easily be a global thing. Let's make sure it's clear this is AzureAuth specific.